views/register.hbs

```diff
@@ -0,0 +1,21 @@
+<section>
+    <div class="signin">
+        <div class="signin__content" tabindex="0">
+            <h2>Signup</h2>
+            <p>We do not share your personal details with anyone.</p>
```

Comment:
Don't hardcode the text in html, move it to a separate translations file. Ideally these translations should be stored server side

Reply:
Done.